src/work-allocation/components/task-assignment/task-assignment.component.ts

```diff
@@ -75,25 +78,67 @@ export class TaskAssignmentComponent implements OnInit {
     return this.pCaseworkers;
   }
 
+  /**
+   * Gets the private logged in user id
+   */
+  public get userId(): string {
+    return this.pUserId;
+  }
+
+  /**
+   * Sets the private logged in user id
+   */
+  public set userId(value: string) {
```

Comment:
I still think this circumvents the `setupUserId()` method, which isn't ideal behaviour. It provides a way for the ID of user stored in the session and the `userId` in here to get out of sync, which could easily lead to problems. My prior comment was to highlight this risk, rather than to suggest using getters and setters instead. 🤷

Reply:
Will remove setter to decrease risk